src/dispatch/organization/service.py

```diff
@@ -131,7 +131,8 @@ def update(
     """Updates an organization."""
     organization_data = organization.dict()
 
-    update_data = organization_in.dict(skip_defaults=True, exclude={"banner_color"})
+    # we exclude the organization's name because the organization db schema is named after it
+    update_data = organization_in.dict(skip_defaults=True, exclude={"name", "banner_color"})
```

Comment:
I think we should remove organization "name" from the update model here: https://github.com/Netflix/dispatch/blob/master/src/dispatch/organization/models.py#L57

Otherwise, the swagger documentation will say that the field is updateable but we will simply ignore it we would be confusing.

Reply:
Ah, good point. Didn't think about that. Will revert the changes on the last commit and update the Pydantic model.